cosmo_tester/framework/test_hosts.py

```diff
@@ -856,8 +856,16 @@ def _save_manager_logs(self):
             'Attempting to save manager logs for test:  {0}'.format(test_path))
         logs_dir = os.path.join(os.path.expanduser(logs_dir), test_path)
         util.mkdirs(logs_dir)
-        for instance in self.instances:
-            self._save_logs_for_instance(instance, logs_dir)
+        for i, instance in enumerate(self.instances):
+            instance_deleted = getattr(instance, "deleted", None)
+            self._logger.debug('Instance {0} deleted status: {1}'.format(
```

Comment:
What's the log msg suppose to say?

Reply:
Not very informative apparently since the next `if` clause is very simple to understand and debugged in the logs. Removing this one.